### PR TITLE
Update VCWG Charter to include W3C CCG input.

### PR DESCRIPTION
**Pull request by msporny**

### CODE REVIEW

**Comment by plehegar** [comment]
https://github.com/w3c/charter-drafts/pull/239/commits/2aa098404fb07da7b0cc5b1bf121f6a021a2bcaf removes the changes done by the previous commits, effectively making zero changes in this PR (no files are being changed). Can you remove that commit for the PR?

File: verifiable-credentials-2019.html [hunk 1]
```diff
@@ -173,18 +173,13 @@ <h3>
             Other Deliverables
           </h3>
           <dl data-query="active-specifications">
-            <dt><a href="https://www.w3.org/TR/vc-use-cases/">Verifiable Credentials Use Cases</a></dt>
+            <dt><a href="https://www.w3.org/TR/vc-use-cases/">Verifiable Claims Use Cases</a></dt>
```
Comment (plehegar) [inline verifiable-credentials-2019.html]: This change (and others) is undoing previous commits.

**Review by plehegar — CHANGES_REQUESTED** [review]

File: verifiable-credentials-2019.html (hunk 1 above)
Comment (msporny) [reply verifiable-credentials-2019.html]: Thanks, was wondering why the diff was showing up blank.

**Comment by msporny** [comment]
> 2aa0984 removes the changes done by the previous commits, effectively making zero changes in this PR (no files are being changed). Can you remove that commit for the PR?

Done, sorry got drawn into a random meeting, back and fixed. I checked the diff, looks clean AFAICT.

Please merge when you're ready.

**Pull request merged**